config/locales/server.en.yml

```
@@ -1049,6 +1049,7 @@ en:
     all: "Don’t send me any mail from %{sitename}"
     different_user_description: "You are currently logged in as a different user than the one we emailed. Please log out, or enter anonymous mode, and try again."
     not_found_description: "Sorry, we couldn't find this unsubscribe. It’s possible the link in your email is too old and has expired?"
+    user_not_found_description: "Sorry, we couldn't find a user for this unsubscribe. It's possible the account no longer exists."
```

I find it odd to use "unsubscribe" as a noun here. I see it's used in the same way in the string above, but it doesn't sound quite right. Maybe: 

```suggestion
    user_not_found_description: "Sorry, we couldn't find a user for this subscription. You are probably attempting to unsubscribe an account that no longer exists."
```

And for the `not_found_description` string, we could try with: "Sorry, we couldn't find that subscription. It's possible..."